host-interaction/hardware/memory/manipulate-linux-process-memory.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: manipulate Linux process memory
+    namespace: host-interaction/hardware/memory
+    author: alyssa.rahman@fireeye.com
+    scope: file
+    examples:
+      - 3db3e55b16a7b1b1afb970d5e77c5d98
+  features:
+    - and:
+      - os: linux
+      - or:
+        - function:
+          - or:
+            - api: malloc
+            - api: memcpy
+            - api: memmove
+            - api: memset
+        - import: free
```

Comment:
i think we should prefer API functions to imports, unless the feature makes more sense at the file level. when we use an API feature, capa can tell us exactly which function has the capability. when we use imports, we just get a general sense of the capabilities.

so, lets make this a function (or basic block) scoped rule and rely on API features